docs/branches.md

```diff
@@ -11,23 +11,28 @@ pass a branch's name to commands that want a revision as argument. For example,
 `jj branch list` to list branches and `jj branch` to create, move, or delete
 branches. There is currently no concept of an active/current/checked-out branch.
 
-## Remotes
-
-Jujutsu records the last seen position on each remote (just like Git's
-remote-tracking branches). You can refer to these with
-`<branch name>@<remote name>`, such as `jj new main@origin`.
-
-A remote branch can be associated with a local branch of the same name. It's
-sometimes called a tracking branch. When you pull from a remote, any changes
-compared to the current record of the remote's state will be propagated to the
-tracking local branch. Let's say you run `jj git fetch --remote origin` and the
-remote's "main" branch has moved so its target is now ahead of the local record
-in `main@origin`. That will update `main@origin` to the new target. It will also
-apply the change to the local branch `main`. If the local target had also moved
-compared to `main@origin` (probably because you had run `jj branch set main`),
+## Remotes and tracked branches
+
+Jujutsu records the last seen position of a branch on each remote (just like
+Git's remote-tracking branches). This record is updated on every `jj git fetch`
+and `jj git push` of the branch. You can refer to the remembered remote branch
+positions with `<branch name>@<remote name>`, such as `jj new main@origin`. `jj`
+does not provide a way to manually edit these recorded positions.
+
+A remote branch can be associated with a local branch of the same name. This is
+called a **tracked branch**. When you pull a tracking branch from a remote, any
+changes compared to the current record of the remote's state will be propagated
+to the tracking local branch.
+
+**Details:** Let's say you run `jj git fetch --remote origin`, and and `jj`
```

Comment:
I'd like to somehow mark this as "skip on first reading". `**Details**` is the best idea I've had for this so far.

Reply:
Use a [alert](https://github.com/orgs/community/discussions/16925#discussion-4085374) here?

Reply:
I'd love to, but unfortunately it seems that MkDocs uses a different syntax: https://squidfunk.github.io/mkdocs-material/reference/admonitions/. I should check whether they have an FR to support GitHub syntax.

Reply:
On second thought, maybe I'll just use their syntax.

Reply:
The callout now looks like this on the website:

![image](https://github.com/martinvonz/jj/assets/4123047/45d79e7a-35db-492e-a227-004b7e3383c1)

Reply:
See https://martinvonz.github.io/jj/v0.13.0/github/#adding-new-commits, to see how the alert in [github.md](https://github.com/martinvonz/jj/blob/main/docs/github.md) is rendered

Reply:
Interesting. I hadn't scrolled down enough on the link you provided, and thus haven't realized GitHub has two syntaxes: `> [!NOTE]` and just `> **Note**`. The latter does look OK in MkDocs, though it's annoying it can't have a period afterwards.

I have a few minor problems with the latter syntax:

- I'm not sure "Note" is the right type of a callout for this. GitHub documents it as "Highlights information that users should take into account, even when skimming.", and I was going for the opposite effect. I also don't see a better option among what GitHub provides, the closest is "Tip".
- GitHub considers the `> **Note**` syntax obsolete, though they don't seem in a hurry to remove it.

So, I think I still prefer what I have now? I'm not sure.

Reply:
I don't mind either syntax, but it would be nice to have it displayed uniformly in Markdown and on the website. In the end we'll need to chose one of them.